tests/test_android.py

```diff
@@ -9,25 +9,12 @@
 from platformdirs.android import Android
 
 
-@pytest.mark.parametrize(
-    "params",
-    [
-        {},
-        {"appname": "foo"},
-        {"appname": "foo", "appauthor": "bar"},
-        {"appname": "foo", "appauthor": "bar", "version": "v1.0"},
-        {"appname": "foo", "appauthor": "bar", "version": "v1.0", "opinion": False},
-    ],
-    ids=[
-        "no_args",
-        "app_name",
-        "app_name_with_app_author",
-        "app_name_author_version",
-        "app_name_author_version_false_opinion",
-    ],
-)
 def test_android(mocker: MockerFixture, params: Dict[str, Any], func: str) -> None:
```

Comment:
Why touch the android tests?

Reply:
Refactored out common parameters into a fixture, since I didn't see the point in pasting the same mark decorator.

Reply:
Ah, ok.